public/tab/tabConfig/popUpSignin.html

```diff
@@ -0,0 +1,77 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+        <style type="text/css">
+            .row {
+                padding: 8px 0px;
+            }
+    
+            .row input {
+                border: none;
+                border-bottom: 1px solid #4E9DC7;
+                background-color: transparent;
+                margin: 0 5px;
+                padding: 3px 5px;
+            }
+    
+            .row input:hover {
+                background-color: rgba(0,0,0,0.05);
+            }
+
+            .row input:active {
+                background-color: rgba(0,0,0,0.1);
+            }
+    
+            .row .label {
+                margin-bottom: 5px;
+            }
+    
+            .primary-button {
+                background: #4C509D;
+                border: 0.125rem solid;
+                border-color: transparent;
+                border-radius: 0.1875rem;
+                color: #FFFFFF;
+                cursor: pointer;
+                font: inherit;
+                height: 2rem;
+                min-width: 6rem;
+                padding: 0.25rem;
+                white-space: nowrap;
+                font-family: 'Segoe UI', Tahoma, Helvetica, Sans-Serif;
+            }
+        </style>    
+</head>
+<!--Purpose of this login html page is to show fake authentication when user clicks on SignIn Action -->
+<!--This login page will do nothing, except when user will click on Login page it will post back to Main controller -->
```

Comment:
Main controller?

Reply:
So I think I main issue I have with this sample is that by reading it I'm not sure how I can apply it to a "real" scenario. In an actual flow, what is the magic number supposed to map to?

Reply:
adding magic number thing to sample app is to show how we can pass the value from popup to bot, e.g. scenario where the popup is showing a list of items and the user wants to update the selected list item in Bot card.

Reply:
I think you need to explain this better, either in the sample or whatever supporting docs we have for this sample. Put yourself in the shoes of a developer-- do you understand what this is showing, and how you're supposed to apply it in your own app?